debug.json

```diff
@@ -0,0 +1,6 @@
+{
+	"postConditions": true,
```

Comment:
Argh, tabs!!!

Also, I think we might want to name this something like `debug.example.json` and then add `debug.json` to the .gitignore file so that git isn't always squawking about your non-default debug config file.